README.md

```diff
@@ -26,6 +26,22 @@ There are no dependencies.
 
 Below is a sample playbook:
 
+##### For deploying agents to Non-Multi-Tenancy Deep Security Manager
+```yaml
+- hosts: all
+  roles:
+    - role: deep-security.deep-security-agent
+      action: deploy
+      dsm_agent_download_hostname: app.deepsecurity.trendmicro.com
```

Comment:
I think we should not use DSaaS url as example, I think use the default OP setting, ex. 4119 and 4120 port will be better to fit users environment.